src/get-serve-d.ts

```diff
@@ -0,0 +1,32 @@
+import downloadRelease from "@terascope/fetch-github-release"
+import { join, dirname, basename, extname } from "path"
+import { remove, ensureDir } from "fs-extra"
+import decompress from "decompress"
+import decompressTarxz from "decompress-tarxz"
+
+// function to download serve-d binaries from GitHub
+export async function getServeD() {
+  const distFolder = join(dirname(__dirname), "dist")
+  await remove(distFolder)
+  await ensureDir(distFolder)
+  const assets = ((await downloadRelease(
+    "Pure-D",
+    "serve-d",
+    distFolder,
+    undefined,
+    undefined,
+    true
+  )) as unknown) as string[]
+  for (const asset of assets) {
```

Comment:
should it really download all assets for a given release?

Reply:
No, I am going to fix this.

Reply:
Fixed